packages/blocks/src/store/selectors.js

```diff
@@ -823,22 +824,47 @@ export const hasChildBlocksWithInserterSupport = ( state, blockName ) => {
 };
 
 /**
- * DO-NOT-USE in production.
- * This selector is created for internal/experimental only usage and may be
- * removed anytime without any warning, causing breakage on any plugin or theme invoking it.
+ * Determines if any of the block type's attributes have
+ * the content role attribute.
+ *
+ * @param {Object} state         Data state.
+ * @param {string} blockTypeName Block type name.
+ * @return {boolean} Whether block type has content role attribute.
  */
-export const __experimentalHasContentRoleAttribute = createSelector(
+export const hasContentRoleAttribute = createSelector(
```

Comment:
Can we make this one a private selector instead? It feels like a frameworky selector that is not really useful for folks outside core.

Reply:
I agree it seems like internal implementation.

However, is that legit? Making something previously (nominally) public into a private API? The `deprecation` prompt would need to explain it's been moved to a private API. Maybe via the `hint` field?

Reply:
It's legit if we keep the __experimental one as deprecated yes.

Reply:
Ok done.